models/labels/cex/labels_cex_ethereum.sql

```diff
@@ -0,0 +1,496 @@
+{{config(alias='cex_ethereum')}}
+
+SELECT blockchain, address, name, category, contributor, source, created_at, updated_at
+FROM (VALUES
+    -- Binance, Source: https://etherscan.io/accounts/label/binance
+    (array('ethereum'),'0x3f5ce5fbfe3e9af3971dd833d26ba9b5c936f0be', 'Binance 1', 'cex', 'hildobby', 'static', timestamp('2022-08-28'), now())
```

Comment:
if this model is specific to ethereum, do we need to allow an array value for ```blockchain``` since it'll always be ethereum to match the model?

Reply:
just noticed #1513 which i think clarifies my question here. while this PR may not use it, others will in the other PR, then later combined into one object

Reply:
Yes exactly, for some more examples you can look at # 1474 as well :)